external/avs/Makefile

```diff
@@ -206,7 +219,8 @@ depend: .depend
 clean:
 	$(call DELFILE, .built)
 	$(call CLEAN)
-	find . -name "*.o" -exec rm -rf {} \;
+	$(foreach FILE, $(CXXOBJS), $(call DELFILE, $(FILE)))
+	rm -f $(TOPDIR)/../tools/fs/contents/AlexaClientSDKConfig.json
```

Comment:
@zhouxinhe 
zhou, would you add command for $ (Q)~?
The user does not need to know.

Reply:
Done, thanks.